libpod/runtime_img.go

```diff
@@ -155,9 +283,22 @@ func (r *Runtime) PullImage(imgName string, allTags bool, signaturePolicyPath st
 		if manifest.Annotations == nil || manifest.Annotations["org.opencontainers.image.ref.name"] == "" {
 			return errors.Errorf("error, archive doesn't have a name annotation. Cannot store image with no name")
 		}
-		images = append(images, manifest.Annotations["org.opencontainers.image.ref.name"])
-	} else {
-		images = append(images, imgName)
+		pullInfo, err := getPullStruct(srcRef, manifest.Annotations["org.opencontainers.image.ref.name"], r.store)
+		if err != nil {
+			return err
+		}
+		pullStructs = append(pullStructs, pullInfo)
+	} else if srcRef != nil && srcRef.Transport().Name() == DirTransport {
+		// supports pull from a directory
+		image := splitArr[1]
+		if image[:1] == "/" {
+			image = image[1:]
```

Comment:
Is this supposed to remove a leading or trailing `/`? The two lines works on the opposite ends.

Reply:
Its removing a leading "/", for the case where the directory path can be something like `/tmp/my-dir`

Reply:
I’m sorry, this is fine. I really don’t know what I was thinking.